README.rst

```diff
@@ -148,6 +148,10 @@ that you install the following packages using a good package-management system (
 
 * h5py
 
+* pylab
```

Comment:
I'm a little confused -- I think that pylab is typically part of matplotlib

Reply:
good catch forgot I took this out above